matchbox/fixture_test.go

```diff
@@ -0,0 +1,245 @@
+package matchbox
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"net"
+
+	"google.golang.org/grpc"
+
+	"github.com/coreos/matchbox/matchbox/rpc"
+	"github.com/coreos/matchbox/matchbox/server"
+	"github.com/coreos/matchbox/matchbox/storage"
+)
+
+type FixtureServer struct {
+	Store    storage.Store
+	Server   *grpc.Server
+	Listener net.Listener
+}
+
+func NewFixtureServer(s storage.Store) *FixtureServer {
+	return &FixtureServer{
+		Store: s,
+	}
+}
+
+func (s *FixtureServer) Start() {
+	var err error
+	s.Listener, err = net.Listen("tcp", "127.0.0.1:0")
```

Comment:
The server is being [stop](https://github.com/coreos/terraform-provider-matchbox/pull/11/files#diff-86d4f962b11c27d9dcc6dfea3fa53a4fR15), is just a race condition.